packages/medusa/src/services/order-edit.ts

```diff
@@ -382,4 +386,36 @@ export default class OrderEditService extends TransactionBaseService {
 
     return orderEdit
   }
+
+  async deleteItemChange(
+    orderEditId: string,
+    itemChangeId: string
+  ): Promise<void> {
+    return await this.atomicPhase_(async (manager) => {
+      const itemChange = await this.orderEditItemChangeService_.retrieve(
+        itemChangeId,
+        { relations: ["order_edit"] }
+      )
+      const orderEdit = itemChange.order_edit
```

Comment:
nit(non-blocking): could consider avoiding the join here and instead do:
```
await this.retrieve(orderEditId, { select: ["id"] }) // to validate that orderEditId exists
```

Reply:
and then just verify `itemChange.order_edit_id === orderEditId`

Reply:
You are right 😊

Reply:
is it worth checking the existence of the order edit since if we delete a change if the order edit does not exists we probably want to still delete it?

Reply:
would [that](https://github.com/medusajs/medusa/pull/2241/commits/41086c669abdc20fd2d2f044926edf9fcb61d669) be enough?

Reply:
This is great!